### ISSUE TITLE
Not functional in smartthing

### ISSUE THREAD

**Issue by SuperJimbo** [tobycth3/Sesame#5]
when adding the lock as a device, smartthings returns the following error:

Error
500: Internal Server Error
URI
/device/save
Reference Id
006cbc87-e2a8-4e9f-84f9-c401fe484b12

**Comment by kahulinut** [comment]
I have the same problem. Did you ever solve this?

**Comment by eesung** [comment]
I followed this instruction from the [candy house community page](https://community.candyhouse.co/t/sesame-smartthings-integration/1046/40).

--------------------------------
 //_Find this around line 255 in the code_
def login() { 
log.info "Executing 'login'"
//state.auth = settings.apikey <==comment out this
state.auth = 'replacewithyoursesameAPIkey' <== add this
// get device ID
deviceID()
log.info state
}
------------------------------------
So basically you need to replace 'settings.apikey' with your actual API key. This at least got me pass that 500 internal error, however I am still getting API Failed warning in Smartthings app so more digging needed.
